rails-commenting.rb

```diff
@@ -9,22 +9,27 @@
 # ---1)
 class BlogPostsController < ApplicationController
```

Comment:
This one is referencing the BlogPostsController and how it will inherit from ApplicationController.